android/build.gradle

```diff
@@ -47,4 +51,12 @@ subprojects {
             }
         }
     }
-}
\ No newline at end of file
+    afterEvaluate {project ->
+        if (project.hasProperty("android")) {
```

Comment:
fix the dependency version problem, e.g. in `react-native-secure-store` the compiledSdkVersion is 27, which will leads a compiling error.